README.md

```diff
@@ -64,6 +64,7 @@ Producer
 * `USER_KEY` - the user's private key
 * `LOG_LEVEL` - logging level  
 * `PRODUCER_ACKS` = acknowledgement level
+* `HEADERS` = custom headers for sending messages
```

Comment:
as for your implementation you should mention that it's a list separated by commas of key=value

Reply:
Will do!

Reply:
Will do!